product_configurator_wizard/wizard/product_configurator.py

```diff
@@ -783,7 +786,8 @@ def action_config_done(self):
         if self.product_id:
             self.product_id.write({
                 'attribute_value_ids': [(6, 0, self.value_ids.ids)],
-                'value_custom_ids': [(6, 0, custom_vals)]
+                'value_custom_ids': map(lambda cv: (2, cv), self.product_id.value_custom_ids.ids) + \
+                                        self.product_id.product_tmpl_id.encode_custom_values(custom_vals),
```

Comment:
Custom vals needs to be modified, with the "encode"

This encode is returning (0, 0, {...}) and I didn't want to create an alternate hook.  The (2, x) is needed to remove the existing links before adding the newly encoded ones, and not get duplicates.

(6, 0 is only available for (6, 0, [ids]), and not for (6, 0, {values}), and even if it works, does not quite make sense with a one2many, only a many2many.

According to the docs,

> For a many2many field, a list of tuples is expected.
> Here is the list of tuple that are accepted, with the corresponding semantics ::
>                  (0, 0,  { values })    link to a new record that needs to be created with the given values dictionary
>                  (1, ID, { values })    update the linked record with id = ID (write *values* on it)
>                  (2, ID)                remove and delete the linked record with id = ID (calls unlink on ID, that will delete the object completely, and the link to it as well)
>                  (3, ID)                cut the link to the linked record with id = ID (delete the relationship between the two objects but does not delete the target object itself)
>                  (4, ID)                link to existing record with id = ID (adds a relationship)
>                  (5)                    unlink all (like using (3,ID) for all linked records)
>                  (6, 0, [IDs])          replace the list of linked IDs (like using (5) then (4,ID) for each ID in the list of IDs)
> 
> For a one2many field, a lits of tuples is expected.
>               Here is the list of tuple that are accepted, with the corresponding semantics ::
> 
>                  (0, 0,  { values })    link to a new record that needs to be created with the given values dictionary
>                  (1, ID, { values })    update the linked record with id = ID (write *values* on it)
>                  (2, ID)                remove and delete the linked record with id = ID (calls unlink on ID, that will delete the object completely, and the link to it as well)